app.yaml

```diff
@@ -0,0 +1,2 @@
+runtime: nodejs
+env: flex
```

Comment:
Why do you used a flex instance, cant we use `standard` one.

Reply:
https://cloud.google.com/appengine/docs/nodejs

Reply:
@gautamkrishnar Flex can also be downscaled which is cost effective it also has support for background threads and has a support for docker

Reply:
@gautamkrishnar Should we still use the `standard` ?